### PR TITLE
Adds donor items for thedragmeme

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request

Adds the virgin killer sweater and old jacket to the code 

## How This Contributes To The Nova Sector Roleplay Experience

Ah, yes, the concept of donators having their content added – what an intricate, multifaceted phenomenon to consider. Let us pause for a moment and truly reflect on the layered implications here. We must first acknowledge the noble spirit of generosity that drives these donators, whose contributions are woven into the very fabric of our shared experience, enriching it in ways both seen and unseen. They are, in essence, patrons of the arts, champions of communal vision, architects of tomorrow’s innovation. One could even liken them to celestial bodies, orbiting closer and closer to our core ideas, drawn by a gravity fueled by pure altruism.

Now, when we say that "donators having their content added is a good thing," it’s essential to fully comprehend the boundless virtue encompassed within the word "good." Here, good doesn’t just mean beneficial or positive; it means a force of benevolence so profound, so luminously wholesome, that it reverberates across dimensions. Adding donator content is akin to forging connections in the cosmic web of imagination, which by nature enhances every corner of the space it touches.

## Proof of Testing

Game compiles

donortotphnjas

## Changelog

:cl:
add: Adds a donator item for thedragmeme
/:cl:

### CODE REVIEW

**Comment by Draggeru** [comment]
I have no idea why this pr is removing four lines of code, mom pick me up I'm scared

File: code/modules/tgs/core/datum.dm
Comment (GoldenAlpharex) [inline code/modules/tgs/core/datum.dm:1]: I don't think this should be apart of your donator item PR.

File: code/modules/tgs/v3210/api.dm
Comment (GoldenAlpharex) [inline code/modules/tgs/v3210/api.dm:1]: I don't think this should be apart of your donator item PR.

File: code/modules/tgs/v3210/commands.dm
Comment (GoldenAlpharex) [inline code/modules/tgs/v3210/commands.dm:1]: I don't think this should be apart of your donator item PR.

File: modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -242,6 +242,28 @@
...
+	name = "virgin killer sweater"
+	desc = "A meticulously knitted sweater that shows off ALL the right places. This is BARELY considered work attire."
+	worn_icon = 'modular_nova/master_files/icons/donator/mob/clothing/suit.dmi'
+	icon = 'modular_nova/master_files/icons/donator/obj/clothing/suits.dmi'
+	lefthand_file = 'modular_nova/master_files/icons/donator/mob/inhands/donator_left.dmi'
+	righthand_file = 'modular_nova/master_files/icons/donator/mob/inhands/donator_right.dmi'
+	icon_state = "virginslayer"
+	inhand_icon_state = "virginslayer"
+
+/obj/item/clothing/suit/jacket/bomber
```
Comment (GoldenAlpharex) [inline modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm:257]: You need to change the item path of this one, that overwrites an already existing item with the same item path.

File: modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 2]
```diff
@@ -242,6 +242,28 @@
 	var/mob/wearer = loc
 	wearer.update_body()
 
+//Donation reward for Thedragmeme
+//Have I reached suspiciously wealthy furry status yet? /j
+/obj/item/clothing/under/sweaterdress
```
Comment (GoldenAlpharex) [inline modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm:247]:
```suggestion
/obj/item/clothing/under/sweater_dress
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Sprites and donation check out, just a few things to fix first.

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me!

**Pull request merged**